sip/sip-003-peer-network.md

```diff
@@ -872,21 +916,58 @@ receiver for the blocks it has via the control-plane,
 and then queries individual blocks and microblocks on the data-plane.
 
 On the control-plane, the sender builds up a locally-cached inventory of which
-blocks the receiver has.  To do so, the sender and receiver do the following:
-
-1.  The sender creates a `GetBlocksInv` message for a range of blocks it wants,
+blocks the receiver has.  To do so, the sender and receiver execute a two-phase
+protocol to synchronize the sender's view of the receiver's block inventroy.
+First, the sender downloads the receiver's knowledge of PoX reward cycles,
+encoded as a bit vector where a `1` in the _ith_ position means that the
+receiver is certain about the status of the PoX anchor block in the _ith_ reward
```

Comment:
Should we talk about "finality", instead of "certainty"?

Reply:
I don't think so.  The PoX bit vector represents the node's belief about the state of a reward cycle, irrespective of whether or not the belief accurately reflects the true state of the system.  The node can revise its PoX bit vector at any future point in time -- it is never "final," because a subsequent Bitcoin reorg can invalidate it.  Moreover, even if we control for Bitcoin reorgs, PoX bit vector propagation runs as an anti-entropy protocol, so there's nothing "final" about any knowledge a node purports to have about the true state of the system -- the node can be denied full knowledge of PoX state for an arbitrarily long amount of time.